docs/configuration.md

```diff
@@ -226,7 +226,7 @@ A three node cluster could be minimally configured as follows:
 
     data-dir = "/home/pilosa/data"
     bind = "node1.pilosa.com:10101"
-    gossip-port = 12000
+    gossip-port = 12001
```

Comment:
Again, this may be confusing as is, but I also think it's confusing to imply that all nodes in a cluster have to use a different gossip-port when that isn't the case. This example cluster has different host names, so presumably they are on three different machines.

Reply:
I agree, the changes in this PR makes things more confusing. I will close this PR for now and think about how we can convey that information in a non-confusing way. Maybe a separate example?